src/local/index.js

```diff
@@ -1,6 +1,7 @@
 import async from 'async'
-import clone from 'lodash.clone'
+// import clone from 'lodash.clone'
```

Comment:
I think you can remove this line. Object spread can replace the `clone` function